app/src/main/java/chat/rocket/android/activity/MainActivity.java

```diff
@@ -51,7 +52,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
   @Override
   protected void onPause() {
     if (presenter != null) {
-      presenter.release();
+//      presenter.release();
```

Comment:
This is really bad since we'll not clean up the subscriptions made. I can't accept this change.